packages/trader/src/Modules/Trading/Components/Elements/purchase-button.tsx

```diff
@@ -119,8 +119,8 @@ const PurchaseButton = ({
                 'btn-purchase--multiplier': is_multiplier,
                 'btn-purchase--multiplier-deal-cancel': has_deal_cancellation,
                 'btn-purchase--turbos': is_turbos,
-                'btn-purchase--1__vanilla-opts': index === 0 && is_vanilla,
-                'btn-purchase--2__vanilla-opts': index === 1 && is_vanilla,
+                'btn-purchase--has-bottom-gradient-green': index === 0 && (is_accumulator || is_vanilla || is_turbos),
+                'btn-purchase--has-bottom-gradient-red': index === 1 && (is_vanilla || is_turbos),
```

Comment:
@kate-deriv  Turbos are not needed at this line, right? They have only one button (no second red button).
Also, as the best practice suggests, I would recommend avoiding colors in classnames. Instead of 'green' and 'red' it would be better to put 'first' and 'second'.
I personally prefer text over digits but I can see that above classnames with digits ('btn-purchase--1' and 'btn-purchase--2') are already used for old trade types, so you might wanna continue using them and simply add '__has-bottom-gradient' to them. :)

Reply:
Turbos do have both buttons (green and red), screenshots from production are pinned. Among mentioned contracts only Acc have one green button.
I agree with renaming suggestion, will apply option with digits in order to stick to previous className, thank you)
<img width="1182" alt="Screenshot 2024-03-12 at 9 50 33 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/db35774e-6de4-4bc7-8091-d3e44c0be7bc">
<img width="1182" alt="Screenshot 2024-03-12 at 9 50 37 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/1c7e2c16-bc87-4421-896c-8b60e554bc2f">

Reply:
thank you :)